com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -456,33 +467,35 @@ UnityInputProto Exchange(UnityOutputProto unityOutput)
                 QuitCommandReceived?.Invoke();
                 return message.UnityInput;
             }
-            catch (RpcException rpcException)
```

Comment:
Reorganized this to avoid duplicating some of the logic (e.g. `m_IsOpen = false`). Not sure if we have a preference; it's not mentioned in the style guide.